### DISCUSSION TITLE
load texture from web

### DISCUSSION THREAD

**Discussion by arrddon** [cables-gl/cables_docs#432] [Answers & Questions] [answered]
hello, 
Searching for way to get texture data, from web image URL.

I succeed load Image from webURL to DIV element's background image.
But to change material's texture in Cables.gl, I have to transform 'Image from web' to CGL's texture data. 

Tried to edit Texture OP's original code, changing 'File URL' to 'Web URL', it doesn't work.
Can I have some infomation about 'CGL.texture.load' ?

Need some help! thank you for reading.

**Comment by pandrr** [comment]
texture op transforms "image from web" to a texture

what is the problem ? or what do you want to achieve ?

**Comment by pandrr** [comment, marked as answer]
you can put a url into the field and it will try to load that image

which will probably not work because of https://cables.gl/docs/4_export_embed/cors/cors

**Reply by arrddon** [reply]
Thank You! Great help for me.
